docs/config.md

```diff
@@ -26,7 +26,7 @@ for more details.
 | Environment variable | Description | Default value |
 |-|-|-|
 | `OTEL_DOTNET_AUTO_INTEGRATIONS_FILE` | List of bytecode instrumentations JSON configuration filepaths, delimited by the platform-specific path separator (`;` on Windows, `:` on Linux and macOS). For example: `%ProfilerDirectory%/integrations.json` | |
-| `OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS` | Comma-separated list of traces source instrumentations you want to enable. |  |
+| `OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS` | Comma-separated list of traces source instrumentations you want to enable. | all available instrumentations |
```

Comment:
Can you remove setting `OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS` from `run-example.sh` and all `launchSettings.json` example files? And test if the examples are still working as expected.